### ISSUE TITLE
google.com (mobile Chrome version) - 'All filters' menu blocked by Fanboy's Annoyance list

### ISSUE THREAD

**Issue by Rhys-T** [easylist/easylist#15067]
### List the website(s) you're having issues:

- `https://www.google.com/search?q=example`

### What happens?

On the mobile Chrome version of the Google search results page, if you swipe the row of filter buttons (just below the search field) to the left as far as it will go, there's a menu labeled 'All filters'. When the Fanboy's Annoyance list is on, that menu seemingly refuses to open when I tap it. There's no visual change whatsoever to indicate that the page even received the touch/click.

I originally ran into this issue on Firefox for Android. But there's a catch if you're trying to test it there: Normally, Google serves a different, simplified mobile version to Firefox, and that version doesn't even have those filter buttons or the 'All filters' menu. To get the full mobile version in Firefox, you'll need to spoof Chrome's User-Agent string, e.g. by installing the 'Google Search Fixer' extension[^fixer] like I did.

[^fixer]: `https://addons.mozilla.org/en-US/android/addon/google-search-fixer/`

I have confirmed that the same problem also happens in desktop Chrome, if I use the devtools to simulate a mobile device. That may be an easier way of testing things on that particular mobile version of Google. (I can't test it in the real Android Chrome, because it doesn't support extensions, and I don't currently have any of the relevant forks installed.)

### List Subscriptions you're using:

The default ones enabled in uBlock Origin, plus:
- AdGuard Annoyances
- Fanboy's Annoyance (disabling this one makes the problem go away)
- uBlock filters – Annoyances

<details>
<summary>The full list of lists I'm using, in case it matters</summary>

- Built-in (6/6)
  - uBlock filters
  - uBlock filters – Badware risks
  - uBlock filters – Privacy
  - uBlock filters – Quick fixes
  - uBlock filters – Resource abuse
  - uBlock filters – Unbreak
- Ads (1/3)
  - EasyList
- Privacy (1/3)
  - EasyPrivacy
- Malware domains (1/3)
  - Online Malicious URL Blocklist
- Annoyances (3/7)
  - AdGuard Annoyances
  - Fanboy’s Annoyance
  - uBlock filters – Annoyances
- Multipurpose (1/2)
  - Peter Lowe’s Ad and tracking server list

I have a few more in my main browser - but just those when I was testing in desktop Chrome.

</details>

### Your settings

When I originally found it:
- OS/version: Android 13
- Browser/version: Firefox 110.0.1
- Adblock Extension/version: uBlock Origin 1.47.0

Testing on desktop:
- OS/version: macOS 10.15.7
- Browser/version: Google Chrome 110.0.5481.100 → Just updated to 110.0.5481.177 and confirmed it again.
- Adblock Extension/version: uBlock Origin 1.47.2

### Other details:

The filter that's blocking it seems to be on this line: https://github.com/easylist/easylist/blob/cbb775f8e6c3ad017eacc09a2d92eeeacb037956/fanboy-addon/fanboy_notifications_specific_hide.txt#L109

As far as I can tell, it was added to block some sort of mobile-app message, but the id `#stUuGf` is apparently getting used for other kinds of popup too - including this one. I haven't seen the message it was intended to block[^haventseen], so I can't offer any guidance on how to make it more specific.

[^haventseen]: Probably because it _was_ blocking it. Although I don't remember seeing it _before_ I turned the various Annoyances lists on, either - but that's been a while, so…

<details>
<summary>Screenshot, using desktop Chrome devtools</summary>

<img width="612" alt="Screenshot of a Google search results page for 'example', showing the non-functional 'All filters' menu." src="https://user-images.githubusercontent.com/108157737/221128251-145fe1bb-080e-44d0-b93b-ecc571a8cc5d.png">

(Note that the 'Add to Home screen' message at the bottom is a separate issue that I didn't even see until I started testing in desktop Chrome. It seems to be a custom element of type `agsa-home-screen-icon-banner`. I can file a separate issue for it if you want.)

What it should look like when opened:

<img width="612" alt="Screenshot of the same page, now dimmed out and half-covered by a menu titled 'Filter'. The visible options include 'All', 'Images', 'Videos', 'Books', 'News', 'Maps', 'Shopping', 'Flights', and 'Finance'." src="https://user-images.githubusercontent.com/108157737/221129805-82d7d3c0-9045-468f-9c21-facf175bc230.png">

</details>

[AdGuard seems to have the same filter in their Annoyances list](https://github.com/AdguardTeam/AdguardFilters/blob/fe3d4be8757d796ceb291f9f1a7ba4ab83a8242e/AnnoyancesFilter/MobileApp/sections/mobile-app_specific.txt#L2435-L2436) - but only on iOS, not in the uBO version that I'm using. It probably needs to be reported there too, but I don't have an iOS device to test with, so I haven't technically had an issue with theirs and can't include the required screenshot of it happening there.
